tools/dev_console/static/index.html

```
@@ -351,52 +353,80 @@ <h2>Base Labels</h2>
           </div>
 
           <div id="base-label-summary" class="health-summary"></div>
-          <section class="base-label-workspace">
+          <section class="tool-body base-label-workspace">
             <aside>
               <input id="base-label-filter" type="search" placeholder="Filter labels">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)